extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -953,7 +954,11 @@ export class SqlDatabaseTree {
 			});
 		}
 		await this._instanceTable.setDataValues(instanceTableValues);
-		await this._databaseTable.setDataValues(this._databaseTableValues);
+		if (this._model.resumeAssessment && this._model.savedInfo.closedPage >= 2) {
+			this._databaseTable.setDataValues(this._model.savedInfo.migrationDatabases);
```

Comment:
should this be 'await'-ed, I belive setDataValues is async

Reply:
I would encourage you to enable the no-floating-promises for this extension to avoid these kinds of potential issues. See https://github.com/microsoft/azuredatastudio/commit/d774548951aa28dd05f8d209dbd77feac3775508 for an example